kalliope/core/MainController.py

```diff
@@ -38,6 +38,13 @@ def __init__(self, brain=None):
 
         # create an order listener object. This last will the trigger callback before starting
         self.order_listener = OrderListener(self.analyse_order)
+
+        if self.settings.random_on_ready_answers is not None:
+            Say(message=self.settings.random_on_ready_answers)
+        elif self.settings.random_on_ready_sounds is not None:
+            random_sound_to_play = self._get_random_sound(self.settings.random_on_ready_sounds)
```

Comment:
Yes, actually the ConfigurationChecker to be only used for the brain. So, you can just raise an exception in the settingLoader id the file does not exist.
And yes, you can do it for both, thx.